container/view.go

```diff
@@ -312,7 +285,7 @@ func (v *memdbView) getNames(containerID string) []string {
 func (v *memdbView) GetID(name string) (string, error) {
 	s, err := v.txn.First(memdbNamesTable, memdbIDIndex, name)
 	if err != nil {
-		return "", err
+		return "", errdefs.System(err)
 	}
 	if s == nil {
 		return "", ErrNameNotReserved
```

Comment:
FWIW; we probably should think about making `ErrNameNotReserved` and `ErrNameReserved` a proper `errdef` as well (perhaps "conflict" for `ErrNameReserved` ?); I left that for a later exercise for now 😅